### ISSUE TITLE
Update the storagenode rollout confluence page

### ISSUE THREAD

**Issue by profclems** [storj/storagenode-docker#8]
- Document the release procedure 
- Update storagenode rollout confluence page
- Assign which team or role (prod-owner?)

Link to storagenode rollout page: https://storjlabs.atlassian.net/wiki/spaces/OPS/pages/138084357/Storagenode

**Comment by iglesiasbrandon** [comment]
Hey @profclems is this ticket still relevant and worth doing?

**Comment by ifraixedes** [comment]
The [storagenode rolllout confluence page](https://storjlabs.atlassian.net/wiki/spaces/OPS/pages/138084357/Storagenode) is already updated.

**Comment by profclems** [comment]
Yes @ifraixedes but we will still need to add documentation on how/when we will be releasing the docker images

**Comment by ifraixedes** [comment]
@profclems I thought so, but I wanted you to provide the response because I wasn't 100% sure. I only mentioned that one of the things listed is already done, to let @iglesiasbrandon know.
